src/implementations/twig/components/blockquote/blockquote.story.js

```diff
@@ -1,28 +1,61 @@
-import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl/storybook-addon-notes';
 import withCode from '@ecl/storybook-addon-code';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
 
 import defaultData from '@ecl/specs-component-blockquote/demo/data';
 import blockquote from './blockquote.html.twig';
 import notes from './README.md';
 
-const prepareQuote = (data) => {
-  data.citation = text('citation', data.citation, tabLabels.required);
-
-  data.author = text('author', data.author, tabLabels.required);
+const getArgTypes = () => {
+  const argTypes = {};
+  argTypes.author = {
+    name: 'Author',
+    type: { name: 'string', required: true },
+    description: 'Author of the citation',
```

Comment:
for some reasons, I don't see the description on storybook
![image](https://user-images.githubusercontent.com/25579221/102186732-5620f280-3eb3-11eb-95f5-250c824a3f50.png)
Is it the same for you?

Reply:
Thks forgot to return `argTypes` in getArgTypes function 👍